org.mixedrealitytoolkit.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -71,7 +71,7 @@ public static InteractionModeManager Instance
         public void InitializeControllers()
         {
             controllerMapping.Clear();
-            foreach (XRController xrController in FindObjectUtility.FindObjectsByType<XRController>())
+            foreach (XRBaseController xrController in FindObjectUtility.FindObjectsByType<XRBaseController>())
```

Comment:
May want to include a bit more in the commit message about why this was needed? What broke, etc?

Reply:
@ghazen-ml 

I did include the following in the commit message:
'''
* Also fixes an issue with the "Init Controllers" type lookup within
  InteractionModeManager.css to find XRBaseControllers.
'''
The issue was that the legacy MRTK Controller classes derive from XRBaseControllers, not XRController, so the "Init Controllers" button wasn't finding anything.

Reply:
I could add a little more verbosity...

Reply:
@ghazen-ml Commit message augmented to explain the addition to this commit regarding this change.